mkdocs.yml

```diff
@@ -136,6 +141,7 @@ plugins:
         - code_of_conduct.md
         - license.md
   - markdown-exec
+  - mkdocs-nav-weight
```

Comment:
What `mkdocs-nav-weight` does is that it sorts pages by the value of `weight` that can be specified in the front matter, e.g.

```txt
weight: 5
---
```